lms/static/scripts/frontend_apps/components/BasicLtiLaunchApp.js

```diff
@@ -381,6 +387,44 @@ export default function BasicLtiLaunchApp({ clientRpc }) {
           <ErrorDisplay error={/** @type {Error} */ (error)} />
         </Dialog>
       )}
+      {errorState === 'canvas-file-not-found-in-course' && (
+        <Dialog
+          initialFocus={focusedDialogButton}
+          title="Hypothesis couldn't find the file in the course"
+          role="alertdialog"
+          buttons={[
+            <Button
+              buttonRef={focusedDialogButton}
+              className="BasicLtiLaunchApp__button"
+              disabled={fetchCount > 0}
+              key="retry"
+              label="Try again"
+              onClick={refetchContentUrl}
+            />,
+          ]}
+        >
+          <p>This might have happened because:</p>
+
+          <ul>
+            <li>The file has been deleted from Canvas</li>
+            <li>The course was copied from another course</li>
+          </ul>
+
+          <p>
+            To fix the issue,{' '}
+            <a
+              target="_blank"
+              rel="noreferrer"
+              href="https://web.hypothes.is/help/fixing-a-broken-canvas-file-link/"
+            >
+              edit the assignment and re-select the file
+            </a>
+            .
+          </p>
+
+          <ErrorDisplay error={/** @type {Error} */ (error)} />
+        </Dialog>
+      )}
```

Comment:
The frontend changes here look fine. There is some duplication across the various error dialogs but either myself or one of the other front-end devs can address that in a follow-up.